apps/server/src/modules/user/repo/user.entity.ts

```diff
@@ -253,7 +253,11 @@ export class User extends BaseEntityWithTimestamps {
 		let setOfPermissions = new Set(permissions);
 
 		// This exclusion is necessary because of possible double roles (e.g., admin and teacher). Then the higher role should keep its permissions.
-		if (roles.some((role) => role.name === RoleName.ADMINISTRATOR || role.name === RoleName.SUPERHERO)) {
+		if (roles.some((role) => role.name === RoleName.SUPERHERO)) {
+			return setOfPermissions;
+		}
+		if (roles.some((role) => role.name === RoleName.ADMINISTRATOR)) {
+			setOfPermissions.add(Permission.SCHOOL_LIST_ROOM_MEMBERS);
```

Comment:
I dont know if these additions are correct in the entity file.
If this is a school role it should appear within the correct roles files. e.g. `backup/setup/roles.json`

Not sure about the roles.ts file above. What about the `authorization-context-params.ts` file which might need that permission?

maybe @Metauriel  and @hoeppner-dataport can comment on where its needed. But here it somehow feels wrong.

Reply:
Martin is correct. in this file you only need to add permissions that are not on the role per default, but are added to it based on a feature setting on the school. In this case since you have it in the seed data and migration nothing more needs to be done here. (but good work finding it :P )